ui/css/treeherder-navbar.css

```diff
@@ -474,3 +486,7 @@ fieldset[disabled] .btn-view-nav-closed.active {
 .dropdown-item:hover {
     background-color: #d3d3d34d;
 }
+
+.dropdown-item > span {
```

Comment:
This doesn't seem to make any difference.  I don't think we need this CSS styling, do we?

Reply:
This was to force the Treestatus and Pushlog dropdown items to stay black despite having visited the page. It doesn't apply now that I'm removing the superfluous <span>s elsewhere in this review. I can change the rule to `a.dropdown-item` to get the same effect.

Reply:
superfluous SPANs*. Silly github parsing...

Reply:
OK, cool.  sounds good.  :)